### DISCUSSION TITLE
Delete my account

### DISCUSSION THREAD

**Discussion by greatlarrymartin** [community/community#67928] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

If I delete my account, it can recover or not after deleting it if 'yes' then how?

**Comment by web-emran** [comment]
I think you can do. you need to contact Github team the can recovered.

**Comment by hrahman12** [comment, marked as answer]
Please see the github docs for deleting your account, before you delete it make sure to save your backups, wikis, forks etc.. Once deleted all data is erased. https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account

**Reply by alicebonin6479** [reply]
I don't have anything on my account I didn't know that I was signing up for developer

**Reply by Bladedc67** [reply]
I don't have anything on my account didn't know I was signing up with developer

**Reply by sstamef** [reply]
I canno longer operate my office 365 account which is the same as github. 
I must have created github in error  as I am 76 yo and have no need

**Reply by sstamef** [reply]
Are you able to hel me please .

**Reply by Alaa12-abd** [reply]
اريد حذف حسابي alaazfod@gmail.com 
لم اكن اعرف اسجل في مطور

**Reply by ok-lw-soltan** [reply]
مرحبا اريد حذف حسابي لكن لايمكنني بسبب انني في دولة محظورة اريد حذفه

**Reply by ghost** [reply]
I can't delete this. Please cancel my account ASAP

**Reply by ghost** [reply]
Can anyone help me

**Comment by TexanToTheBone** [comment]
somehow I ended up with two accounts...please delete the patnshirley@gmail.com account.
thanks

**Reply by Greenboss69** [reply]
Same here idk how I ended up with two accounts  I want both them deleted plz

**Comment by Diamondshine1998** [comment]
Delete my account permanently

**Comment by Nuwan-Srimal** [comment]
Same here idk how I ended up

**Comment by uniman88** [comment]
Delete my account permanently

**Reply by haven2a** [reply]
اريد حذف حسابي

**Comment by Cardosofiles** [comment]
If you delete your GitHub account, it’s generally permanent and cannot be recovered. However, there are a few things to consider:

Recovery Time Window: GitHub states that once an account is deleted, the username and any associated content are typically gone immediately. You may be able to contact GitHub Support within a very short time window after deletion, but account recovery is not guaranteed. GitHub usually warns users before final deletion.

Data Backup: Before deleting your account, GitHub recommends that you download and save any repositories or data you may want to keep. Once an account is deleted, all repositories, gists, issues, and pull requests associated with it are also permanently removed.

Repositories Owned by Organizations: If you are the owner of an organization, transferring ownership of repositories and assigning new organization owners is crucial before deleting your account, as organization-owned repositories can remain active if properly transferred.

If you accidentally delete your account and need assistance, you can reach out to [GitHub Support](https://support.github.com/) immediately after deletion, though recovery is generally limited or unavailable.

**Comment by ghost** [comment]
delete my account permanently
